### PR TITLE
Fix from #179

### PR DESCRIPTION
**Pull request by fuentesloic**

### CODE REVIEW

**Comment by fuentesloic** [comment]
@vinyll this patch don't solve the data on dev :)

File: static/css/home.css [hunk 1]
```diff
@@ -13,31 +13,34 @@ h2 {
...
 
-section:first-child > div > img {
-  width: 100%;
-  filter: brightness(50%);
-  min-height: 50rem;
-}
-section:first-child > div > h1,
-section:first-child > div > h2 {
+section:first-child {
+background: linear-gradient(0deg, rgba(0,0,0,0.5), rgba(0, 0, 0, 0.5)), url('../images/home.jpg');
```
Comment (vinyll) [inline static/css/home.css]: spaces after comas :p

**Review by vinyll — DISMISSED** [review]
quick comment for fine art :)

**Comment by fuentesloic** [comment]
Crap I was so close :)

**Review by vinyll — APPROVED** [review]

**Pull request merged**